diff -urN aa/gnome-chemistry-utils-0.2.3/docs/reference/html/Makefile.am gnome-chemistry-utils-0.2.3/docs/reference/html/Makefile.am
--- aa/gnome-chemistry-utils-0.2.3/docs/reference/html/Makefile.am	2003-08-27 21:55:47.000000000 +0200
+++ gnome-chemistry-utils-0.2.3/docs/reference/html/Makefile.am	2004-02-12 15:50:27.627443248 +0100
@@ -1,10 +1,10 @@
 HTMLDIR = $(datadir)/doc/gchemutils/reference/html
 
 install-data-local:
-	$(mkinstalldirs) $(HTMLDIR); \
+	$(mkinstalldirs) $(DESTDIR)$(HTMLDIR); \
   for i in *.html *.png *.css; do \
 	echo '-- Installing '$$i ; \
-	$(INSTALL_DATA) $$i $(HTMLDIR); \
+	$(INSTALL_DATA) $$i $(DESTDIR)$(HTMLDIR); \
   done
 
 dist-hook: 
